<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowContent</class>
 <widget class="QMainWindow" name="MainWindowContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>770</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>770</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>770</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="channelGroupBox">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>40</y>
      <width>251</width>
      <height>701</height>
     </rect>
    </property>
    <property name="title">
     <string>Channels</string>
    </property>
    <widget class="QListView" name="channels">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>611</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="addChannelButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>660</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Channel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeChannelButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>660</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Channel</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="UserModsTabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>40</y>
      <width>1011</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="userTab">
     <attribute name="title">
      <string>Users</string>
     </attribute>
     <widget class="QTableView" name="users">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>981</width>
        <height>601</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::DashLine</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="usernamLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>635</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>635</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="reasonLabel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>635</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reason:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="reasonLineEdit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>635</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="banUserButton">
      <property name="geometry">
       <rect>
        <x>824</x>
        <y>635</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ban User</string>
      </property>
     </widget>
     <widget class="QPushButton" name="unbanUserButton">
      <property name="geometry">
       <rect>
        <x>914</x>
        <y>635</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Unban User</string>
      </property>
     </widget>
     <widget class="QLabel" name="urlReasonlabel">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>635</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>URL:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="urlReasonLineEdit">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>635</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="connectionIcon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLogin"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>Login...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
